roles/lib_openshift/src/ansible/oadm_certificate_authority.py

```diff
@@ -0,0 +1,49 @@
+# pylint: skip-file
+# flake8: noqa
+
+def main():
+    '''
+    ansible oadm module for ca
+    '''
+
+    module = AnsibleModule(
+        argument_spec=dict(
+            state=dict(default='present', type='str',
```

Comment:
Yeah, I'm fine with removing it as well.  Consistency.  I guess an update is possible so 'present' represents create and update.